invenio_app_ils/config.py

```diff
@@ -187,6 +191,10 @@ def _(x):
         "schedule": timedelta(hours=3),
         "args": [("record-view-agg", "file-download-agg")],
     },
+    "clean_past_exceptions": {
```

Comment:
I am afraid this is not descriptive enough, just reading it I am thinking to exceptions as Python errors. What about something like:
```suggestion
    "clean_locations_past_closures_exceptions": {
```